backend/src/openbeheer/zaaktype/tests/test_zaaktype_detail.py

```diff
@@ -179,82 +151,82 @@ def test_retrieve_zaaktype(self):
+                                "afleidingswijze": "afgehandeld",
                                 "datumkenmerk": "",
                                 "einddatumBekend": False,
                                 "objecttype": "",
-                                "procestermijn": None,
                                 "registratie": "",
+                                "procestermijn": None,
                             },
-                            "catalogus": "http://localhost:8003/catalogi/api/v1/catalogussen/ec77ad39-0954-4aeb-bcf2-6f45263cde77",
-                            "eindeGeldigheid": None,
-                            "eindeObject": None,
-                            "indicatieSpecifiek": None,
-                            "informatieobjecttypeOmschrijving": [],
-                            "informatieobjecttypen": [],
-                            "omschrijving": "Lopend",
-                            "omschrijvingGeneriek": "",
                             "procesobjectaard": "",
+                            "indicatieSpecifiek": None,
                             "procestermijn": None,
-                            "resultaattypeomschrijving": "Lopend",
-                            "selectielijstklasse": "https://selectielijst.openzaak.nl/api/v1/resultaten/cc5ae4e3-a9e6-4386-bcee-46be4986a829",
-                            "toelichting": "",
-                            "url": "http://localhost:8003/catalogi/api/v1/resultaattypen/7759dcb7-de9a-4543-99e3-81472c488f32",
-                            "zaaktype": "http://localhost:8003/catalogi/api/v1/zaaktypen/ec9ebcdb-b652-466d-a651-fdb8ea787487",
-                            "zaaktypeIdentificatie": "ZAAKTYPE-2020-0000000001",
-                        },
-                        {
-                            "archiefactietermijn": None,
-                            "archiefnominatie": "",
+                            "catalogus": "http://localhost:8003/catalogi/api/v1/catalogussen/d001f879-496e-4844-8a64-fcdd0eab6b23",
+                            "besluittypen": [],
+                            "besluittypeOmschrijving": [],
+                            "informatieobjecttypen": [],
+                            "informatieobjecttypeOmschrijving": [],
                             "beginGeldigheid": None,
+                            "eindeGeldigheid": None,
                             "beginObject": None,
-                            "besluittypeOmschrijving": [],
-                            "besluittypen": [],
+                            "eindeObject": None,
+                        },
+                        {
+                            "url": "http://localhost:8003/catalogi/api/v1/resultaattypen/6052ca60-5062-4887-b0c1-a6c2b4fdec45",
+                            "zaaktype": "http://localhost:8003/catalogi/api/v1/zaaktypen/df34e352-ecc7-46cd-a18c-7998fac92716",
+                            "zaaktypeIdentificatie": "ZAAKTYPE-2025-0000000058",
```

Comment:
I let you deduce from the identificatie how many attempts this has taken... 🙈 😬

Reply:
At least you didn't have to `docker compose down && docker compose up` before each attempt any more 🙂